.github/workflows/node.yml

```diff
@@ -58,14 +58,6 @@ jobs:
         run: |
           npm ci
           npm run build --if-present
-          npm run stats --if-present
-
-      - name: Send bundle stats to RelativeCI
```

Comment:
you want this?

Reply:
That's why we have to be careful :see_no_evil: